src/server/api/controllers/orders.controller.js

```diff
@@ -0,0 +1,30 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getOrders = async () => {
+  return knex('orders');
+};
+
+const getOrderByUserId = async (userId) => {
+  if (!userId) {
+    throw new HttpError(
+      'Bad request. Order ID must be an integer and larger than 0',
+      400,
+    );
+  }
+
+  try {
+    const orders = await knex('orders').select('orders').where({ userId });
```

Comment:
PLease check your endpoint .This  querry return error not orders by user_id

Reply:
Yes, since there is no userId on the orders table. It should be user_id as in the endpoint: api/orders?user={user_id}